e2e/dractions/retry.go

```diff
@@ -68,14 +68,32 @@ func waitDRPCPhase(ctx types.Context, namespace, name string, phase ramen.DRStat
 		}
 
 		currentPhase := drpc.Status.Phase
-		if currentPhase == phase {
-			log.Debugf("drpc \"%s/%s\" phase is %q in cluster %q", namespace, name, phase, hub.Name)
+		phaseMet := currentPhase == phase
+
+		currentProgression := drpc.Status.Progression
+		progressionCompleted := currentProgression == ramen.ProgressionCompleted
+
+		if phaseMet && progressionCompleted {
+			log.Debugf("drpc \"%s/%s\" phase is %q and progression is completed in cluster %q",
+				namespace, name, phase, hub.Name)
```

Comment:
This does not "add more logs" but changes the semantics to waitDRPCPhaseCompleted(). The commit message promises to "add more logs" which contradicts this change.

The way we use this check is to wait until the phase changes to the wanted phase, and move on to the next check. In failover and relocate we call waitDRPCReady() which waits on the conditions and lastGroupSyncTime.

I don't think we want to wait for completion in waitDRPCPhase(). If you think that we must wait for completion before moving to next step, we need to add the Completed progression check in waitDRPCReady().

In waitDRPCReady we already wait for multiple conditions and we can add also the progression to the check, and add it to the log showing the state when we time out.